grails-app/migrations/core/2019-03-26-new-ethereum-call-module.groovy

```diff
@@ -0,0 +1,30 @@
+package core
+databaseChangeLog = {
+	changeSet(author: "jtakalai", id: "2019-03-26-new-ethereum-call-module") {
+		insert(tableName: "module") {
+			column(name: "id", valueNumeric: 1150)
+			column(name: "version", valueNumeric: 0)
+			column(name: "category_id", valueNumeric: 1001) // Ethereum
+			column(name: "implementing_class", value: "com.unifina.signalpath.blockchain.SendEthereumTransaction")
+			column(name: "name", value: "EthereumSend")
```

Comment:
If this module replaces an existing module (`EthereumCall`?), should we use the same name to avoid throwing off users? (Unless the old name is wrong/misleading of course). But for a module that calls a function on a smart contract, "send" isn't ideal IMO.

Reply:
renamed new module 'EthereumCall' and old one 'EthereumCall (Old)'

Reply:
If the old module is to be hidden, then sure, the name could be the same. Can our search / auto-complete deal with that?

EthereumCall is not a very good name though... maybe EthereumFunctionCall? Or just EthereumFunction?